mmcv/runner/hooks/profiler.py

```diff
@@ -10,14 +10,18 @@
 
 @HOOKS.register_module()
 class ProfilerHook(Hook):
-    """Profiler to analyze perfromance during training and inference.
+    """Profiler to analyze perfromance during training.
 
     PyTorch Profiler is a tool that allows the collection of the performance
-    metrics during the training and inference. More details on Profiler can
-    be found at
+    metrics during the training. More details on Profiler can be found at
     https://pytorch.org/docs/1.8.1/profiler.html#torch.profiler.profile
 
     Args:
+        by_epoch (bool): Profile performance by epoch or by iteration.
+            Default: True.
+        profile_iters (int): Number of iterations for profiling.
+            If ``by_epoch=True``, num_iters indicates epochs, otherwise it
```

Comment:
1. num_iters -> profile_iters
2. Indicate that they are the first xxx iterations/epochs at the beginning of the training.